src/IDelegateRegistry.sol

```diff
@@ -10,40 +10,61 @@ pragma solidity ^0.8.20;
 interface IDelegateRegistry {
     /// @notice Delegation type
     enum DelegationType {
-        NONE,
         ALL,
         CONTRACT,
-        ERC721,
         ERC20,
+        ERC721,
         ERC1155
     }
 
-    /// @notice Info about a single delegation, used for onchain enumeration
-    struct DelegationInfo {
+    /// @notice Standardizes storage positions of delegation data
+    enum StoragePositions {
+        delegate,
+        vault,
+        rights,
+        contract_,
+        tokenId,
+        balance
+    }
+
+    /// @notice Struct for returning arbitrary delegations
+    struct Delegation {
         DelegationType type_;
+        bool enable;
```

Comment:
We only want to return the `enabled` delegations, if false then it should be cleared from storage altogether. Think this struct is used primarily on the enumeration front, maybe a way to simplify by removing this boolean

Reply:
`enabled` is never stored in storage. All this struct is used for is a structure in memory for returning arbitrary delegations in the `getDelegationsFor` functions.. which should all return as valid anyway. So maybe not needed :)